bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/AuthFilter.java

```diff
@@ -181,7 +183,7 @@ protected void deactivate() {
         }
     }
 
-    private SecurityContext authenticateBearerToken(String token) throws AuthenticationException {
+    public SecurityContext authenticateBearerToken(String token) throws AuthenticationException {
         if (token.startsWith(API_TOKEN_PREFIX)) {
             UserApiTokenCredentials credentials = new UserApiTokenCredentials(token);
             Authentication auth = userRegistry.authenticate(credentials);
```

Comment:
While it is not related to your changes, this call should call `org.openhab.core.auth.AuthenticationManager` and not user registry itself.